test/treeshake/utils/format-size.js

```diff
@@ -0,0 +1,7 @@
+// used in report-size.yml
+import prettyBytes from 'pretty-bytes';
```

Comment:
Maybe just this function, instead of a new dependency?

```javascript
export function formatBytes(bytes: number, decimals = 2): string {
	if (bytes === 0) return '0 Bytes';

	const k = 1000;
	const dm = decimals < 0 ? 0 : decimals;
	const sizes = ['Bytes', 'KB', 'MB'];

	const i = Math.floor(Math.log(bytes) / Math.log(k));

	return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
}
```

Reply:
Less dependencies is good 👍

Reply:
Thanks for the function @donmccurdy! Updated.